ocs_ci/ocs/resources/cloud_manager.py

```
@@ -131,6 +133,15 @@ def __init__(self):
         except ClusterNotInSTSModeException:
             setattr(self, "azure_sts_client", None)
 
+        try:
+            setattr(
+                self,
+                "self_ref_mcg_client",
+                cloud_map["SELF_REF_MCG"](),
+            )
```

Ruff (0.15.15)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)